opentracing-shim/src/main/java/io/opentelemetry/opentracingshim/SpanShim.java

```diff
@@ -214,10 +232,39 @@ static Attributes convertToAttributes(Map<String, ?> fields) {
       } else if (value instanceof Boolean) {
         attributesBuilder.put(booleanKey(key), (Boolean) value);
       } else {
-        attributesBuilder.put(stringKey(key), value.toString());
+        if (isExceptionEvent) {
+          switch (key) {
+            case Fields.ERROR_KIND:
+              key = "exception.type";
```

Comment:
we should definitely make these three keys be statically initialized (the corresponding AttributeKey instances) to avoid extra allocations in here.

Reply:
Or, better yet, use the ones already defined in the SemanticAttributes class in the semconv module.

Reply:
`SemanticAttributes` is not available in this module, should I add new dependency?

Reply:
My opinion: definitely, we should use the semantic conventions in our shim. @carlosalberto what do you think? It would mean that the shim depends on an alpha module (of course, the shim is also alpha, so that seems fine).

Reply:
Sounds good for now! Hopefully we don't end up reverting that in case the OT shim becomes stable _and_ semantic conventions are not ;)

Reply:
If that happens, we can easily copy the needed keys over to the shim in the interim.

Reply:
done